sonar-core/src/main/resources/org/sonar/core/qualityprofile/db/QualityProfileMapper.xml

```diff
@@ -151,6 +151,14 @@
     </where>
   </select>
 
+  <select id="countProjectsByProfile" resultType="org.sonar.core.qualityprofile.db.QualityProfileProjectCount">
```

Comment:
Default behavior is INNER, but explicit > implicit. I'll also add the where, thanks!